app/proguard-rules.pro

```diff
@@ -1,6 +1,6 @@
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in /Users/scottgo/Library/Android/sdk/tools/proguard/proguard-android.txt
```

Comment:
This feels like a legacy from where something was first generated. Most users won't have a `/Users/scottgo` directory and it won't be their home directory.